dashboard/test/integration/db_query_test.rb

```diff
@@ -103,7 +103,7 @@ def setup
     sl = create(:script, :with_levels, levels_count: 3).script_levels[2]
     params = {program: 'fake program', testResult: 0, result: 'false'}
 
-    assert_cached_queries(9) do
+    assert_cached_queries(10) do
```

Comment:
This release adds a query to check if a script_level is AI enabled whenever a student submits a project.